pkgs/tools/system/pdisk/default.nix

```diff
@@ -0,0 +1,90 @@
+{ stdenv
+, lib
+, fetchzip
+, fetchpatch
+, installShellFiles
+, libbsd
+, CoreFoundation
+, IOKit
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pdisk";
+  version = "0.9";
+
+  src = fetchzip {
+    url = "https://opensource.apple.com/tarballs/pdisk/pdisk-${lib.versions.minor version}.tar.gz";
```

Comment:
It's not super clear to me if the version is supposed to be `0.9`, or just `9`, or something else.

Case for 0.x:
- https://opensource.apple.com/source/pdisk/pdisk-9/pdisk.html mentions a version 0.8, so there at least existed a 0.x series
- The [README](https://github.com/apple-oss-distributions/pdisk) that's actually included with the downloads says 0.8 dated 2000, which I assume just hasn't been updated
- Checking [`version.h`](https://github.com/apple-oss-distributions/pdisk/blob/49a742e56feed141170c4a555ecd9e531378aeaa/version.h#L50-L51) says 0.9a2 dated 2004
- Running `pdisk --version` says `version 0.9a2 (16 May 2004)`

Case for x:
- https://opensource.apple.com/releases/ only lists a single number, 10.4 is the first release to ship with a `pdisk-5`
- All of the above-linked files have gone basically unchanged since they were introduced to the (publicly-viewable) history: in `pdisk-5`

As for what other distros are doing about this: On [repology](https://repology.org/project/pdisk/versions) the version numbers are either 0.8, 0.8a2 or 0.9.

I don't think it's unimaginable to go for something like `0.9a2-9` either? With `0.9a2` being what the executable identifies itself as and the `-9` whatever Apple's numbers mean to them. I don't know. Happy about opinions on this.

Reply:
Sigh, complicated. How about we just keep it like it is right now.

Reply:
Fine by me.